tools/mo/openvino/tools/mo/convert_impl.py

```diff
@@ -888,6 +894,17 @@ def _convert(cli_parser: argparse.ArgumentParser, framework, args, python_api_us
                 decoder = get_pytorch_decoder(args['input_model'], parse_input_shapes(args), example_inputs, args.get("input"))
                 args['input_model'] = decoder
                 args["framework"] = "pytorch"
+            if model_framework == "paddle":
+                example_inputs = None
+                if 'example_input' in args and args['example_input'] is not None:
+                    example_inputs = args['example_input']
+
+                example_outputs = None
+                if 'example_output' in args and args['example_output'] is not None:
+                    example_outputs = args['example_output']
+                pdmodel = convert_paddle_to_pdmodel(args['input_model'], example_inputs, example_outputs)
+                args['input_model'] = pdmodel
+                args['framework'] = "paddle"
```

Comment:
please avoid paddle specific things in the common MO py files

Reply:
> please avoid paddle specific things in the common MO py files

fixed, use a pre-defined variable instead.